aarch64/src/kernel.ld

```diff
@@ -0,0 +1,62 @@
+/*
+ * Linker script for R9.
+ */
+
+ENTRY(start)
+
+SECTIONS {
+	/*
+	 * start the kernel at 0x0xffff_8000_0020_0000
+	 * This preserves some RAM between 1MiB and the
+	 * start of the kernel for critical structures.
+	 */
+	/* . = 0xffff800000100000; */
```

Comment:
This is just a copy of the linker script from x86_64? Please do a `git cp` so that we can track the provenance of the file.

Reply:
There is no `git cp`. Hmm. Curious that it didn't notice the duplication.